docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -135,6 +135,17 @@ The "one-time" nature of triggers may result in a small number of edge cases whe
 
 Finally, users should be aware that in some cases proper trigger execution may depend on the timing and order of events with respect to creating nodes, setting properties, creating nodes from secondary properties ("derivative nodes"), etc. The detailed technical aspects of Synapse write operations are beyond the scope of this discussion, but as always it is good practice to test triggers (or other automation) before putting them into practice to ensure they perform as intended.
 
+**Variables:**
+
+Triggers automatically have the Storm variable ``$auto`` populated when they run. The ``$auto`` variable is a dictionary which contains the following keys:
+
+  ``$auto.iden``
+    The identifier of the Trigger.
+
+  ``$auto.type``
+    The type of automation. For a Trigger this value will be ``dmon``.
+
```

Comment:
Do we wanna take this opportunity to plumb and document something like `$auto.opts.tag` (etc) for the other vars ( and leave the existing ones in place for now obvs ) ?

Reply:
We certainly could. Each trigger through typoe would end up with its own opts dict based on what was done. This is also a good opportunity to at least through in a `$form` and `$valu` key so that is universally available across triggers.